docker/travis_test/prepare.sh

```diff
@@ -27,6 +27,7 @@ zypper in -y -C \
        qemu-tools \
        qemu-kvm \
        tar \
+       opencv-devel \
```

Comment:
as much as you enjoy duplicating efforts - I will not merge this. We will only have one docker image